build2.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sh build.sh 2
```

Comment:
Not sure how this helps.  OTOH, it can make some things harder.  E.g. `source build2.sh` behaves differently than `source build.sh`, and `build2.sh` will run `build.sh` in POSIX mode.  So either `source build.sh 2`, or just let the user tough it out with the extra space and type `build.sh 2`.